src/dune_rules/pkg_rules.ml

```diff
@@ -151,20 +244,20 @@ module Paths = struct
     Path.Build.append_local t.extra_sources extra_source
   ;;
 
-  let make package_universe name =
-    let universe_root =
-      match (package_universe : Package_universe.t) with
+  let make pkg_digest universe =
+    let root =
+      match (universe : Package_universe.t) with
+      | Dependencies ctx ->
+        Path.Build.relative
```

Comment:
Isn't there something like `Path.Build.L.relative`? If not, we should introduce something so that it's readable.